custom/icds_reports/sqldata/exports/dashboard_usage.py

```diff
@@ -220,21 +222,21 @@ def get_excel_data(self):
                     user_location_row = None
                     # iterating and getting the db row from matrix
                     for row in location_matrix:
-                        if row[column_index] == user_sql_location:
-                            user_location_row = row
+                        if row[2 * column_index] == user_sql_location:
```

Comment:
Why is it `2x`?
continuing that 2X0 will be 0 only. Is it expected?
@challabeehyv

Reply:
so basically we will have to look for values from these columns ['state_id', 'state_name', 'district_id', 'district_name', 'block_id', 'block_name']...so when we get state_id we look for rows under column 0..so for district_id it will be [2] and for block_id ii will be [4]

Reply:
> @challabeehyv I had left one comment. Not sure if you got the notification or not

replied to it. please check